language/vm/src/access.rs

```diff
@@ -186,6 +186,13 @@ pub trait ScriptAccess: Sync {
     /// Returns the `CompiledScript` that will be used for accesses.
     fn as_script(&self) -> &CompiledScript;
 
+    /// Returns the `ModuleHandle` for `self`.
```

Comment:
not sure if there is a better way to do this than replicating the function in CompileModule but here....
also CompiledScript is going to change (remove) this concept so I feel a bit whatever about this....